src/app.ts

```diff
@@ -19,12 +31,9 @@ export default class App {
     await LoggerManager.mountLogger();
     await CommunicationServiceManager.mountService();
 
-    // Micro Services
-    const accountServiceRESTApi = await AccountServiceManager.createRestAPIServer();
-    this.app.use('/', accountServiceRESTApi);
-
-    const accessTokenServiceRESTApi = await AccesstokenServiceManager.createRestAPIServer();
-    this.app.use('/', accessTokenServiceRESTApi);
+    // loading all microservices
+    const microServicesApi = await this.createRESTApiServer();
```

Comment:
Rename variable to `restAPIServices`. Remove comment - `// loading all microservices`